From c1b0a95419f7ca43405a49b2e639091105ece31b Mon Sep 17 00:00:00 2001
From: Michael W <baddaemon87@gmail.com>
Date: Sat, 7 Sep 2019 18:50:28 +0200
Subject: [PATCH 1/3] CallLog: Get rid of warning

* ContextImpl: Calling a method in the system process
  without a qualified user

Change-Id: Ibc44e240c5f8d9290d53d0ee009120e5b34152bf
---
 src/com/android/server/telecom/CallLogManager.java | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/com/android/server/telecom/CallLogManager.java b/src/com/android/server/telecom/CallLogManager.java
index fe5cfb1c1..6e6136d33 100644
--- a/src/com/android/server/telecom/CallLogManager.java
+++ b/src/com/android/server/telecom/CallLogManager.java
@@ -556,7 +556,8 @@ public final class CallLogManager extends CallsManagerListenerBase {
         Intent callAddIntent = new Intent(ACTION_CALLS_TABLE_ADD_ENTRY);
         callAddIntent.putExtra(CALL_TYPE, callType);
         callAddIntent.putExtra(CALL_DURATION, duration);
-        mContext.sendBroadcast(callAddIntent, PERMISSION_PROCESS_CALLLOG_INFO);
+        mContext.sendBroadcastAsUser(callAddIntent, UserHandle.SYSTEM,
+                PERMISSION_PROCESS_CALLLOG_INFO);
     }
 
     private String getCountryIsoFromCountry(Country country) {
-- 
2.42.1

